symengine/tests/test_sympy_conv.py

```diff
@@ -432,6 +443,100 @@ def test_log():
     assert sympify(sympy.log(x1, y1)) == log(x, y)
 
 
+def test_lambertw():
+    x = Symbol("x")
+    e1 = sympy.LambertW(sympy.Symbol("x"))
+    e2 = LambertW(x)
+    assert sympify(e1) == e2
```

Comment:
This checks sympy->symengine conversions and not the other way around. It would be nice to have that.